tests/traits/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -86,7 +86,9 @@ class ImplementingClass
 
 class ShortImplementingClass
 {
-    use testTrait { A::testFunction insteadof C; }
+    use testTrait {
+        A::testFunction insteadof C;
+    }
```

Comment:
We always add `newline` after `{` (in function, class, closure and etc), let's do this here for consistently output

Reply:
I am wrong here, better use 
```
use testTrait {A::testFunction insteadof C;}
```

Reply:
From PSR-12:
> When using the insteadof and as operators they must be used as follows taking note of indentation, spacing and new lines.

https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#42-using-traits

Reply:
:+1: Which is eaxtly what you implemented now, right?

Reply:
@czosel Yep :+1: